tests/args.py

```diff
@@ -164,6 +169,42 @@ def test_commands_together(self):
         self.assertEqual(args.commands, [["test", "args"], ["restart", "all"]])
 
 
+class TestProfileArguments(unittest.TestCase):
+
```

Comment:
Consider adding a test to ensure that a profile can override the default `component`, `restart` and `host`